src/plugins/graph/utilities/graph-utils.ts

```diff
@@ -140,37 +141,203 @@ export interface IMatchCirclesProps {
 }
 
 export function matchCirclesToData(props: IMatchCirclesProps) {
-
   const {dataConfiguration, enableAnimation, instanceId,
-      dotsElement, pointRadius, pointColor, pointStrokeColor} = props,
-    allCaseData = dataConfiguration.joinedCaseDataArrays,
-    caseDataKeyFunc = (d: CaseData) => `${d.plotNum}-${d.caseID}`,
-    circles = selectCircles(dotsElement);
-  if (!circles) return;
+      dotsElement, pointRadius, pointColor, pointStrokeColor} = props;
+
+  const allCaseData = dataConfiguration.joinedCaseDataArrays;
+  const allCircles = selectAllCircles(dotsElement); //includes both inner and outer circles
+  if (!allCircles) return;
   startAnimation(enableAnimation);
-  circles
-    .data(allCaseData, caseDataKeyFunc)
+
+  initializePoints({
+    selection: allCircles,
+    data: allCaseData,
+    pointRadius,
+    pointColor,
+    pointStrokeColor,
+    getPointColorAtIndex: (index) => pointColor,
+    instanceId
+  });
+  //----------------- Click on Dot -------------------
+  // dotsElement && select(dotsElement).on('click',
+  //   (event: MouseEvent) => {
+  //     const target = select(event.target as SVGSVGElement);
+  //     if (target.node()?.nodeName === 'circle') {
+  //       handleClickOnDot(event, (target.datum() as CaseData).caseID, dataConfiguration.dataset);
+  //     }
+  //   });
+  // dataConfiguration.setPointsNeedUpdating(false);
```

Comment:
With it commented out - shift clicking on a column of dots  on XY Plot selects the case on Data Card
With it in - shift clicking  on a column of dots on XY plot doesn't select the case on Data Card. 
All other functionality is the same. I thought having shift capability would be ideal therefore I commented it out.
I can make more of a commented note on this.

Reply:
Dennis, using the Shift button is in no way part of this story. If anything, I would expect holding down the shift key to allow me to select dots from multiple cases but I don't think it works that way. Please let's make this work consistently with the description.

Reply:
@lbondaryk Okay gotcha - will revert back then.

Reply:
I didn't necessarily think this code should not be commented out--I genuinely didn't understand what it did or why it was removed. I imagine it would be even more mysterious for engineers stumbling upon this commented block in a year from now.